lang/syn/src/parser/error.rs

```diff
@@ -1,3 +1,8 @@
+use syn::{
+    parse::{Parse, Result as ParseResult},
+    Expr, Token,
+};
+
```

Comment:
```suggestion
```

Reply:
nit: we should be consistent with how we import. The rest of the codebase will use different modules on different lines.

Reply:
> The rest of the codebase will use different modules on different lines.

wdym by this? I should separate the imports into separate lines?

Reply:
```rust
use syn::{Expr, Token};
use syn::parse::{Parse, Result as ParseResult};
```